src/main/java/fm/last/moji/impl/MojiImpl.java

```diff
@@ -72,7 +72,7 @@ public void copyToMogile(File source, MojiFile destination) throws IOException {
       outputStream.flush();
     } finally {
       IOUtils.closeQuietly(inputStream);
-      IOUtils.closeQuietly(outputStream);
+      outputStream.close();;
```

Comment:
Can you please paste in the full stack trace thrown with the above code in place?